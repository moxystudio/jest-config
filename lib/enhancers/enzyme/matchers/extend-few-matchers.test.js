'use strict';

it('should extend a few matchers', () => {
    const extendEnzymeMatchers = require('.');

    extendEnzymeMatchers([
        'toBeDisabled',
        'toBeEmptyRender',
        'toExist',
        'toContainReact',
        'toHaveState',
    ]);

    expect(expect(null)).toMatchSnapshot();
});
