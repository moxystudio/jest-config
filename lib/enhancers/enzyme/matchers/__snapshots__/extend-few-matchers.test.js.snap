// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should extend a few matchers 1`] = `
Object {
  "lastCalledWith": [Function],
  "lastReturnedWith": [Function],
  "not": Object {
    "lastCalledWith": [Function],
    "lastReturnedWith": [Function],
    "nthCalledWith": [Function],
    "nthReturnedWith": [Function],
    "toBe": [Function],
    "toBeCalled": [Function],
    "toBeCalledTimes": [Function],
    "toBeCalledWith": [Function],
    "toBeCloseTo": [Function],
    "toBeDefined": [Function],
    "toBeDisabled": [Function],
    "toBeEmptyRender": [Function],
    "toBeFalsy": [Function],
    "toBeGreaterThan": [Function],
    "toBeGreaterThanOrEqual": [Function],
    "toBeInstanceOf": [Function],
    "toBeLessThan": [Function],
    "toBeLessThanOrEqual": [Function],
    "toBeNaN": [Function],
    "toBeNull": [Function],
    "toBeTruthy": [Function],
    "toBeUndefined": [Function],
    "toContain": [Function],
    "toContainEqual": [Function],
    "toContainReact": [Function],
    "toEqual": [Function],
    "toExist": [Function],
    "toHaveBeenCalled": [Function],
    "toHaveBeenCalledTimes": [Function],
    "toHaveBeenCalledWith": [Function],
    "toHaveBeenLastCalledWith": [Function],
    "toHaveBeenNthCalledWith": [Function],
    "toHaveLastReturnedWith": [Function],
    "toHaveLength": [Function],
    "toHaveNthReturnedWith": [Function],
    "toHaveProperty": [Function],
    "toHaveReturned": [Function],
    "toHaveReturnedTimes": [Function],
    "toHaveReturnedWith": [Function],
    "toHaveState": [Function],
    "toMatch": [Function],
    "toMatchInlineSnapshot": [Function],
    "toMatchObject": [Function],
    "toMatchSnapshot": [Function],
    "toReturn": [Function],
    "toReturnTimes": [Function],
    "toReturnWith": [Function],
    "toStrictEqual": [Function],
    "toThrow": [Function],
    "toThrowError": [Function],
    "toThrowErrorMatchingInlineSnapshot": [Function],
    "toThrowErrorMatchingSnapshot": [Function],
  },
  "nthCalledWith": [Function],
  "nthReturnedWith": [Function],
  "rejects": Object {
    "lastCalledWith": [Function],
    "lastReturnedWith": [Function],
    "not": Object {
      "lastCalledWith": [Function],
      "lastReturnedWith": [Function],
      "nthCalledWith": [Function],
      "nthReturnedWith": [Function],
      "toBe": [Function],
      "toBeCalled": [Function],
      "toBeCalledTimes": [Function],
      "toBeCalledWith": [Function],
      "toBeCloseTo": [Function],
      "toBeDefined": [Function],
      "toBeDisabled": [Function],
      "toBeEmptyRender": [Function],
      "toBeFalsy": [Function],
      "toBeGreaterThan": [Function],
      "toBeGreaterThanOrEqual": [Function],
      "toBeInstanceOf": [Function],
      "toBeLessThan": [Function],
      "toBeLessThanOrEqual": [Function],
      "toBeNaN": [Function],
      "toBeNull": [Function],
      "toBeTruthy": [Function],
      "toBeUndefined": [Function],
      "toContain": [Function],
      "toContainEqual": [Function],
      "toContainReact": [Function],
      "toEqual": [Function],
      "toExist": [Function],
      "toHaveBeenCalled": [Function],
      "toHaveBeenCalledTimes": [Function],
      "toHaveBeenCalledWith": [Function],
      "toHaveBeenLastCalledWith": [Function],
      "toHaveBeenNthCalledWith": [Function],
      "toHaveLastReturnedWith": [Function],
      "toHaveLength": [Function],
      "toHaveNthReturnedWith": [Function],
      "toHaveProperty": [Function],
      "toHaveReturned": [Function],
      "toHaveReturnedTimes": [Function],
      "toHaveReturnedWith": [Function],
      "toHaveState": [Function],
      "toMatch": [Function],
      "toMatchInlineSnapshot": [Function],
      "toMatchObject": [Function],
      "toMatchSnapshot": [Function],
      "toReturn": [Function],
      "toReturnTimes": [Function],
      "toReturnWith": [Function],
      "toStrictEqual": [Function],
      "toThrow": [Function],
      "toThrowError": [Function],
      "toThrowErrorMatchingInlineSnapshot": [Function],
      "toThrowErrorMatchingSnapshot": [Function],
    },
    "nthCalledWith": [Function],
    "nthReturnedWith": [Function],
    "toBe": [Function],
    "toBeCalled": [Function],
    "toBeCalledTimes": [Function],
    "toBeCalledWith": [Function],
    "toBeCloseTo": [Function],
    "toBeDefined": [Function],
    "toBeDisabled": [Function],
    "toBeEmptyRender": [Function],
    "toBeFalsy": [Function],
    "toBeGreaterThan": [Function],
    "toBeGreaterThanOrEqual": [Function],
    "toBeInstanceOf": [Function],
    "toBeLessThan": [Function],
    "toBeLessThanOrEqual": [Function],
    "toBeNaN": [Function],
    "toBeNull": [Function],
    "toBeTruthy": [Function],
    "toBeUndefined": [Function],
    "toContain": [Function],
    "toContainEqual": [Function],
    "toContainReact": [Function],
    "toEqual": [Function],
    "toExist": [Function],
    "toHaveBeenCalled": [Function],
    "toHaveBeenCalledTimes": [Function],
    "toHaveBeenCalledWith": [Function],
    "toHaveBeenLastCalledWith": [Function],
    "toHaveBeenNthCalledWith": [Function],
    "toHaveLastReturnedWith": [Function],
    "toHaveLength": [Function],
    "toHaveNthReturnedWith": [Function],
    "toHaveProperty": [Function],
    "toHaveReturned": [Function],
    "toHaveReturnedTimes": [Function],
    "toHaveReturnedWith": [Function],
    "toHaveState": [Function],
    "toMatch": [Function],
    "toMatchInlineSnapshot": [Function],
    "toMatchObject": [Function],
    "toMatchSnapshot": [Function],
    "toReturn": [Function],
    "toReturnTimes": [Function],
    "toReturnWith": [Function],
    "toStrictEqual": [Function],
    "toThrow": [Function],
    "toThrowError": [Function],
    "toThrowErrorMatchingInlineSnapshot": [Function],
    "toThrowErrorMatchingSnapshot": [Function],
  },
  "resolves": Object {
    "lastCalledWith": [Function],
    "lastReturnedWith": [Function],
    "not": Object {
      "lastCalledWith": [Function],
      "lastReturnedWith": [Function],
      "nthCalledWith": [Function],
      "nthReturnedWith": [Function],
      "toBe": [Function],
      "toBeCalled": [Function],
      "toBeCalledTimes": [Function],
      "toBeCalledWith": [Function],
      "toBeCloseTo": [Function],
      "toBeDefined": [Function],
      "toBeDisabled": [Function],
      "toBeEmptyRender": [Function],
      "toBeFalsy": [Function],
      "toBeGreaterThan": [Function],
      "toBeGreaterThanOrEqual": [Function],
      "toBeInstanceOf": [Function],
      "toBeLessThan": [Function],
      "toBeLessThanOrEqual": [Function],
      "toBeNaN": [Function],
      "toBeNull": [Function],
      "toBeTruthy": [Function],
      "toBeUndefined": [Function],
      "toContain": [Function],
      "toContainEqual": [Function],
      "toContainReact": [Function],
      "toEqual": [Function],
      "toExist": [Function],
      "toHaveBeenCalled": [Function],
      "toHaveBeenCalledTimes": [Function],
      "toHaveBeenCalledWith": [Function],
      "toHaveBeenLastCalledWith": [Function],
      "toHaveBeenNthCalledWith": [Function],
      "toHaveLastReturnedWith": [Function],
      "toHaveLength": [Function],
      "toHaveNthReturnedWith": [Function],
      "toHaveProperty": [Function],
      "toHaveReturned": [Function],
      "toHaveReturnedTimes": [Function],
      "toHaveReturnedWith": [Function],
      "toHaveState": [Function],
      "toMatch": [Function],
      "toMatchInlineSnapshot": [Function],
      "toMatchObject": [Function],
      "toMatchSnapshot": [Function],
      "toReturn": [Function],
      "toReturnTimes": [Function],
      "toReturnWith": [Function],
      "toStrictEqual": [Function],
      "toThrow": [Function],
      "toThrowError": [Function],
      "toThrowErrorMatchingInlineSnapshot": [Function],
      "toThrowErrorMatchingSnapshot": [Function],
    },
    "nthCalledWith": [Function],
    "nthReturnedWith": [Function],
    "toBe": [Function],
    "toBeCalled": [Function],
    "toBeCalledTimes": [Function],
    "toBeCalledWith": [Function],
    "toBeCloseTo": [Function],
    "toBeDefined": [Function],
    "toBeDisabled": [Function],
    "toBeEmptyRender": [Function],
    "toBeFalsy": [Function],
    "toBeGreaterThan": [Function],
    "toBeGreaterThanOrEqual": [Function],
    "toBeInstanceOf": [Function],
    "toBeLessThan": [Function],
    "toBeLessThanOrEqual": [Function],
    "toBeNaN": [Function],
    "toBeNull": [Function],
    "toBeTruthy": [Function],
    "toBeUndefined": [Function],
    "toContain": [Function],
    "toContainEqual": [Function],
    "toContainReact": [Function],
    "toEqual": [Function],
    "toExist": [Function],
    "toHaveBeenCalled": [Function],
    "toHaveBeenCalledTimes": [Function],
    "toHaveBeenCalledWith": [Function],
    "toHaveBeenLastCalledWith": [Function],
    "toHaveBeenNthCalledWith": [Function],
    "toHaveLastReturnedWith": [Function],
    "toHaveLength": [Function],
    "toHaveNthReturnedWith": [Function],
    "toHaveProperty": [Function],
    "toHaveReturned": [Function],
    "toHaveReturnedTimes": [Function],
    "toHaveReturnedWith": [Function],
    "toHaveState": [Function],
    "toMatch": [Function],
    "toMatchInlineSnapshot": [Function],
    "toMatchObject": [Function],
    "toMatchSnapshot": [Function],
    "toReturn": [Function],
    "toReturnTimes": [Function],
    "toReturnWith": [Function],
    "toStrictEqual": [Function],
    "toThrow": [Function],
    "toThrowError": [Function],
    "toThrowErrorMatchingInlineSnapshot": [Function],
    "toThrowErrorMatchingSnapshot": [Function],
  },
  "toBe": [Function],
  "toBeCalled": [Function],
  "toBeCalledTimes": [Function],
  "toBeCalledWith": [Function],
  "toBeCloseTo": [Function],
  "toBeDefined": [Function],
  "toBeDisabled": [Function],
  "toBeEmptyRender": [Function],
  "toBeFalsy": [Function],
  "toBeGreaterThan": [Function],
  "toBeGreaterThanOrEqual": [Function],
  "toBeInstanceOf": [Function],
  "toBeLessThan": [Function],
  "toBeLessThanOrEqual": [Function],
  "toBeNaN": [Function],
  "toBeNull": [Function],
  "toBeTruthy": [Function],
  "toBeUndefined": [Function],
  "toContain": [Function],
  "toContainEqual": [Function],
  "toContainReact": [Function],
  "toEqual": [Function],
  "toExist": [Function],
  "toHaveBeenCalled": [Function],
  "toHaveBeenCalledTimes": [Function],
  "toHaveBeenCalledWith": [Function],
  "toHaveBeenLastCalledWith": [Function],
  "toHaveBeenNthCalledWith": [Function],
  "toHaveLastReturnedWith": [Function],
  "toHaveLength": [Function],
  "toHaveNthReturnedWith": [Function],
  "toHaveProperty": [Function],
  "toHaveReturned": [Function],
  "toHaveReturnedTimes": [Function],
  "toHaveReturnedWith": [Function],
  "toHaveState": [Function],
  "toMatch": [Function],
  "toMatchInlineSnapshot": [Function],
  "toMatchObject": [Function],
  "toMatchSnapshot": [Function],
  "toReturn": [Function],
  "toReturnTimes": [Function],
  "toReturnWith": [Function],
  "toStrictEqual": [Function],
  "toThrow": [Function],
  "toThrowError": [Function],
  "toThrowErrorMatchingInlineSnapshot": [Function],
  "toThrowErrorMatchingSnapshot": [Function],
}
`;
